package.json

```diff
@@ -35,17 +42,20 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.16.0",
-    "@opennextjs/cloudflare": "^0.2.1",
+    "@opennextjs/cloudflare": "^0.4.4",
     "@tailwindcss/typography": "^0.5.15",
     "@types/mark.js": "^8.11.12",
     "@types/node": "^20",
+    "@types/pg": "^8.11.10",
```

Comment:
question: we don't use the `pg` package, do we need `@types/pg`?

Reply:
Good catch, accidental carryover from earlier version